### PR TITLE
Multi and batch processing to main 

### PR DESCRIPTION
**Pull request by chrise96**

### CODE REVIEW

**Review by thomasbrockmeier-ams — APPROVED** [review]

**Review by thomasbrockmeier-ams — COMMENTED** [review]
Don't forget to check the automated tests ;)

File: inference_batch.py [hunk 1]
```diff
@@ -119,16 +119,60 @@ def instances_to_coco_json(
...
+    local_file = f"{opt.worker_id}.txt"
+    saClient.download_blob(
+        cname="retrieve-images-input",
+        blob_name=f"{start_date_dag}/{local_file}",
+        local_file_path=local_file,
+    )
+    with open(local_file, "r") as f:
+        pano_ids = [line.rstrip("\n") for line in f]
+
+    if len(pano_ids) < opt.num_workers:
```
[the remark this answers is not in the document]
Comment (epureanudiana) [reply inference_batch.py:133]: Let's maybe do the same as we do in `retrieve_images.py` and raise error only if there are no images.

File: yolov5/detect.py [hunk 2]
```diff
@@ -201,6 +207,44 @@ def main(opt):
...
+    local_file = f"{opt.worker_id}.txt"
+    saClient.download_blob(
+        cname="retrieve-images-input",
+        blob_name=f"{start_date_dag}/{local_file}",
+        local_file_path=local_file,
+    )
+    with open(local_file, "r") as f:
+        pano_ids = [line.rstrip("\n") for line in f]
+
+    if len(pano_ids) < opt.num_workers:
```
Comment (epureanudiana) [inline yolov5/detect.py:221]: Same comment as in `inference_batch.py`

**Review by epureanudiana — APPROVED** [review]
Looks good :)

**Pull request merged**